be/src/olap/compaction.cpp

```diff
@@ -390,11 +391,12 @@ Status Compaction::modify_rowsets() {
         if (_tablet->keys_type() == KeysType::UNIQUE_KEYS &&
             _tablet->enable_unique_key_merge_on_write()) {
             _tablet->tablet_meta()->update_delete_bitmap(
-                    _input_rowsets, _output_rs_writer->version(), _rowid_conversion);
+                    _input_rowsets, _output_rs_writer->version(), _rowid_conversion, &location_map);
         }
 
         RETURN_NOT_OK(_tablet->modify_rowsets(output_rowsets, _input_rowsets, true));
     }
+    _tablet->check_rowid_conversion(_output_rowset, location_map);
```

Comment:
Do we need to add a config here? once user encountered this kind of issue, and we need to trigger the full compaction to fix the data issue.